src/classes/BDI_ManageAdvancedMappingCtrl.cls

```diff
@@ -380,6 +432,7 @@ public class BDI_ManageAdvancedMappingCtrl {
             for(String fieldName : sObjectCastedCustomMetadata.getPopulatedFieldsAsMap().keySet()) {
 
                 Boolean invalidField =
+                    fieldName.contains('__r') ||
```

Comment:
What is exactly is causing us to block the relationship fields - was that not an issue before we started handling cascade delete, or is this a ride-along fix for a separate issue?

Reply:
Ah, neither actually. This is an artifact from when I was trying out another way of cascade deleting. Pushed a new commit reverting this. Thank you!